includes/elements/container.php

```diff
@@ -690,6 +690,13 @@ protected function register_background_overlay_controls() {
 							$background_overlay_selector => '--background-overlay: \'\';',
 						],
 					],
+					'image' => [
```

Comment:
@maork-elementor The background doesn't work in a container (see take for more details).

Reply:
@rotemee 
I just test is with container experiment ON : 

Only background image - worked.
Background image & overlay image - worked. 

<img width="773" alt="Screen Shot 2022-12-01 at 9 03 56" src="https://user-images.githubusercontent.com/100698721/204988028-986e0bf8-e2fa-4d1f-afcc-5e51d2b442e2.png">

This is the CSS for the container overlay : 

<img width="369" alt="Screen Shot 2022-12-01 at 9 04 57" src="https://user-images.githubusercontent.com/100698721/204987690-11adbf90-3247-402e-a8e7-43e801f4265b.png">

What do I need to do to reproduce the problem?

Reply:
@maork-elementor:
"Create a container, and set a background overlay → the background does not load."
Try only a background overlay (without a regular background).

Reply:
You are right, Fixed.